.github/workflows/check_changelog.yml

```diff
@@ -2,8 +2,6 @@ name: Check release notes
 
 on:
   pull_request:
-    branches:
-      - 'main'
```

Comment:
Allow running on any pull request, not just those targeting main - ensures the check applies to stacked pull requests.